src/SSVOpenHexagon/Core/HGScripting.cpp

```diff
@@ -761,6 +761,39 @@ void HexagonGame::initLua_LevelControl()
             "between ``SidesMin`` and ``SidesMax`` inclusively every level "
             "increment.");
 
+    addLuaFn("l_overrideScore", //
+        [this](const std::string& mVar) {
+            try
+            {
+                levelStatus.scoreOverride = mVar;
+                levelStatus.scoreOverridden = true;
+                // Make sure we're not passing in a string
+                lua.executeCode(
+                    "if (type(" + mVar +
+                    ") ~= \"number\") then\n"
+                    "error(\"Score override must be a number value\")\n"
+                    "end");
```

Comment:
A [raw string literal](https://stackoverflow.com/questions/56710024/what-is-a-raw-string) here would help a lot with readability.

Reply:
Copy that.